index.html

```diff
@@ -1,72 +1,225 @@
-        <div class="hero-buttons">
-            <a href="https://sgrewritten.org/download"><button class="btn btn-primary me-2" id="ovrd">
-					<span class="fas me-2 fa-download"></span>Downloads
-				</button></a>
-        </div>
-        <div class="hero-buttons">
-            <a href="https://sgrewritten.org/wiki"><button class="btn btn-outline-secondary" id="b1">
-					<span class="fas me-2 fa-book" id="i1"></span>Documentation
-				</button></a>
-            <a href="https://sgrewritten.org/discord"><button class="btn btn-secondary" id="b2">
-					<span class="fas me-2 fa-comments" id="i2"></span>Support
-				</button></a>
-            <a href="https://sgrewritten.org/bukkitsource"><button class="btn btn-outline-secondary" id="b3">
-					<span class="fas me-2 fa-code" id="i3"></span>Source Code
-				</button></a>
-        </div>
-        <div class="hero-image p-4 shadow-sm rounded" id="otovrd"><img src="https://i.imgur.com/6BMpZKc.png" alt="Gramateria" id="iw1wr" /></div>
-    </section>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/5.0.2/js/bootstrap.min.js"></script>
-    <script src="js/global.js"></script>
-    <script src="js/script.js"></script>
-</body>
-
-<footer>
-    <div style="background-color: rgb(0,0,0, 0.7); padding: 10px 0px 0px 0px !important; bottom: -10px; position: fixed; text-align: center; width:100%;"><span style="color:LightGray;"><b>Temporary website; replacement coming soon!</b></span><br><small><i><p style="color:LightGray;">Background image provided by <a style="color:LightGray;" href="https://unsplash.com/@josiahgascho">Josiah Gascho</a> via <a style="color:LightGray;" href="https://unsplash.com/photos/YnICeUmhzCA">Unsplash</a></p></i></small></div>
-</footer>
-
-</html>
+<!DOCTYPE html>
+<html lang="zh-TW">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <title>The Stargate Rewritten Project</title>
+    <meta name="description" content="The original, and still the best, Minecraft transportation solution!">
+    <link rel="icon" href="img/logo.png" sizes="32x32" type="image/png">
+    <!-- custom.css -->
+    <link rel="stylesheet" href="css/custom.css">
+    <!-- bootstrap.min.css -->
+    <link rel="stylesheet" href="css/bootstrap.min.css">
+    <!-- font-awesome -->
+    <link rel="stylesheet" href="font-awesome-4.7.0/css/font-awesome.min.css">
+    <!-- AOS -->
+    <link rel="stylesheet" href="css/aos.css">
+</head>
+
+<body>
+    <!-- banner -->
+    <div class="jumbotron jumbotron-fluid" id="banner" style="background-image: url(img/bg.jpg);">
+        <div class="container text-center text-md-left">
+            <header>
+                <div class="row justify-content-between">
+                    <div class="col-2">
+                        <img src="img/logo.png" alt="logo">
+                    </div>
+                    <div class="col-6 align-self-center text-right">

```

Comment:
Probably a good idea to include links here to https://sgrewritten.org/discord and https://sgrewritten.org/github